releasing/docker-run.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
I wonder if we should replace this bash script with a Python version?

Reply:
Yes, but not yet. This is going towards https://github.com/umarcor/verible/blob/ci/rework/releasing/docker-run.sh. I think we can change to Python when we achieve that.

Reply:
Okay, SGTM.